app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/notification/AppTpRevokedContentPlugin.kt

```diff
@@ -0,0 +1,71 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.mobile.android.vpn.service.notification
+
+import android.content.Context
+import androidx.core.app.NotificationCompat.Action
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.mobile.android.vpn.AppTpVpnFeature
+import com.duckduckgo.mobile.android.vpn.R
+import com.duckduckgo.mobile.android.vpn.R.drawable
+import com.duckduckgo.mobile.android.vpn.R.string
+import com.duckduckgo.mobile.android.vpn.VpnFeaturesRegistry
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.NotificationContent
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.NotificationPriority
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.NotificationPriority.NORMAL
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.Type
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.Type.REVOKED
+import com.duckduckgo.mobile.android.vpn.ui.notification.NotificationActionReportIssue
+import com.squareup.anvil.annotations.ContributesMultibinding
+import javax.inject.Inject
+
+@ContributesMultibinding(AppScope::class)
+class AppTpRevokedContentPlugin @Inject constructor(
+    private val context: Context,
+    private val vpnFeaturesRegistry: VpnFeaturesRegistry,
+) : VpnReminderNotificationContentPlugin {
+    private val notificationPendingIntent by lazy { getAppTPNotificationPressIntent(context) }
+    private val notificationActionPendingIntent by lazy { getAppTPStartIntent(context) }
+
+    override fun getContent(): NotificationContent? {
+        return NotificationContent(
+            isSilent = true,
+            shouldAutoCancel = null,
+            customViewLayout = R.layout.notification_device_shield_revoked,
+            onNotificationPressIntent = notificationPendingIntent,
+            notificationAction = listOf(
+                Action(
+                    drawable.ic_vpn_notification_24,
+                    context.getString(string.atp_EnableCTA),
+                    notificationActionPendingIntent,
+                ),
+                NotificationActionReportIssue.reportIssueNotificationAction(context),
+            ),
+        )
+    }
+
+    override fun getPriority(): NotificationPriority {
+        return NORMAL
+    }
+
+    override fun isActive(): Boolean {
+        return !vpnFeaturesRegistry.isFeatureRegistered(AppTpVpnFeature.APPTP_VPN)
+    }
```

Comment:
ditto, I think we need to remove.
I have tested the revoke notification and doesn't show. Returning `true` here fixes the issue